### PR TITLE
Adds notice info to mail subject

### LINKED ISSUE #24: Change subject from "Notice: CV submission" to "Placement: CV submission"
It would be better if the subject is changed from `Notice: CV Submission - Company` to `Internship: CV Submission - Company` or `Placement: Urgent - CV Verification`.

This is possible since the first line mentions type:placement or type:internship. This would help reduce confusions amongst people sitting for placements who might confuse notices of internship as being of placements.

### PR DESCRIPTION
**Pull request by athityakumar**
Fixes #24

### CODE REVIEW

**Comment by kaustubhhiware** [comment]
@icyflame mentioned there might be some complexities that need regex to solve. Pinging him for review.

File: hooks.py [hunk 1]
```diff
@@ -41,12 +41,19 @@ def companies_updated(companies):
 
 def notices_updated(notices):
     for notice in notices:
+        if " : PLACEMENT" in notice['text']:
```
Comment (amrav) [inline hooks.py]: It's better to match on the exact string (ie `Type : PLACEMENT`). Also need to handle the case where both internship and placement match.

Comment (icyflame) [reply hooks.py]: Yeah, match the complete string. Also, you need to think about cases where the string might be in the notice but that might not be in the particular type. Might not be very prevalent or common, definitely think about it though.

**Comment by raivivek** [comment]
My 2 cents are that any "notice" associated with placement or internship should be equally "urgent". Since upstream source is not in our control, sudden changes can lead to unpredictable behavior as highlighted by @icyflame

**Pull request closed without merging**